docs/manual/posting-and-listing.rst

```diff
@@ -79,6 +81,43 @@ Any page in a Sphinx_ project can be converted to a post using the following dir
 
       Update directive renders like the updates that are at the end of this post.
 
+.. _posting-front-matter:
+
+Posting with page front-matter
+------------------------------
+
+If you'd prefer to use `page front matter <https://www.sphinx-doc.org/en/1.7/markup/misc.html>`_
+instead of using a directive, you may mark a page as a "blog post" by adding
+the following front-matter at the top:
+
+.. code-block:: rst
+
+   :blogpost: true
+
+``ABlog`` will treat any pages with this front-matter as a blog post. All fields that are
+available to the :ref:`posting directive <posting-directive>` can be given as page-level
+front-matter as well.
+
+.. admonition:: Automatically detect blog posts with a ``glob`` pattern 
```

Comment:
```suggestion
.. admonition:: Automatically detect blog posts with a ``glob`` pattern
```

Reply:
(am installing and running the pre-commit hooks now, sorry about that)

Reply:
It's alright, its new check and normally it's just a whitespace lingering around that's easy to fix